app/views/cities/show.html.erb

```diff
@@ -2,30 +2,30 @@
 
 <style>
```

Comment:
Is having the css inside a view a new Rails thing? Or should we move this to it's own scss file?

Reply:
Naw, this was just quick work from me originally 😄